src/open_company_web/components/topics_columns.cljs

```diff
@@ -112,26 +114,48 @@
       (= min-height thrd-clmn)
       :3)))
 
+(defn get-pinned-layout [pinned-topics columns-num]
```

Comment:
function docstring and possibly some inline comments would be good here. this function is non-trivial.